content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md

```diff
@@ -0,0 +1,335 @@
+  </div>
+
+  <div style="text-align: center">
+![Example portrait, scale 5]({filename}rt-portrait-scale5.jpg)
+  </div>
+
+  <div style="text-align: center">
+![Example portrait, scale 6]({filename}rt-portrait-scale6.jpg)
+  </div>
+
+  <div style="text-align: center">
+![Example portrait, scale 7]({filename}rt-portrait-scale7.jpg)
+  </div>
+
+This kind of transformation is used internally by the "equalizer"
+module, which allows you to increase or decrease the importance of
+each level of detail in the image. While "equalizer" works globally on
+the image, "retouch" allows you to select the level of detail and the
+portion of image you want to work with.
+
+In the retouch module, this corresponds to the "wavelet decompose"
+part of the GUI:
+
+  <div style="text-align: center">
+![Wavelet decompose scales]({filename}rt-scales.png)
+  </div>
+
+This part shows one rectangle per scale (fine-grained on the left,
+coarse-grained on the right). The dark rectangle on the left
+corresponds to the full image, and the white one on the right to the
+residual image, i.e. the image where all other levels of details have
+been removed. By default, darktable always shows the final image, but
+you can visualize the details layers and residual image by clicking
+the "display wavelet scales
+(![Display wavelet scales icon]({filename}rt-display-wavelet-scales.png))
+button. The currently selected image appears with a red rectangle.
+Move the bottom slider to change the number of details layers to use.
+Depending on the zoom level, some details layer may be finer than your
+screen resolution, hence unusable. The grey line above the layers
+shows which layers are visible at the current zoom level.
+
+Let's focus on the spot below the glasses. If we want to completely
+remove it, it's pretty easy to do so with the "heal" tool. Now, what
+happens if we decide that we want to keep it, but reduce it to avoid
+drawing the attention. We can just remove it from the coarse detail
+layer (scale 6 in our example). The spot is no longer visible on the
+residual image, so removing it from the details layer is sufficient.
+The "heal" tool allows us to do that cleanly, but when dealing with
+the details layer, the "fill" and "blur" tools can also give good
+results. Here's the result on scale 6 (before on the left, after on
+the right):
+
+  <div style="text-align: center">
+![Before/after at scale 6]({filename}rt-before-after-scale6.jpg)
+  </div>
+
+The final image will be transformed as follows:
+
+  <div style="text-align: center">
+![Before/after at scale 6 (final image)]({filename}rt-before-after-final.jpg)
+  </div>
+
+Now, we may decide that the healing we did on scale 6 should also be
+applied on scale 5. We can redo the same thing manually, but we can
+also use the top slider, called "merge scale", to automatically
+replicate shapes to multiple layers. Any shape created on the right
+hand side of this slider will be replicated on all finer layers up to
```

Comment:
Internally what happen is that the detail scales are merged before the edits are applied to them, that's why a single edit seems to be applied to each of those scales, but shapes are not replicated.
I was unable to explain this in a clear way in the user manual (or anywhere else), so if this way is more clear to the user, is not only fine with me, but very welcome.

Reply:
Good. Anyway, this article is not meant as a replacement to the user manual, but more as a complement, so it's nice if we have different ways of explaining the same thing.